app/components/collections/admin_component.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Collections
+  # Renders the admin functions for a collection
+  class AdminComponent < ApplicationComponent
+    def initialize(collection:)
+      @collection = collection
+    end
+
+    attr_reader :collection
+
+    def render?
+      helpers.user_with_groups.administrator?
+    end
+
+    def options
+      opts = [
+        ['Select...', 'select']
+      ]
+      opts << ['Decommission collection', edit_collection_decommission_path(collection)] unless collection.head.decommissioned? # rubocop:disable Layout/LineLength
```

Comment:
Don't let the user think they can decommission a thing that's already decommissioned.